simple_history/management/commands/populate_history.py

```diff
@@ -1,7 +1,10 @@
+# -*- coding: utf-8 -*-
 from django.apps import apps
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
```

Comment:
I'd be fine with this PR, except for the linting rule that enforces this change. It seems odd enough to me that I'd want one of the two project leads to approve it.